src/Polly.Core/Simmy/Fault/FaultChaosStrategy.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Polly.Telemetry;
+
+namespace Polly.Simmy.Fault;
+internal class FaultChaosStrategy<T> : MonkeyStrategy<T>
```

Comment:
If this is going to stay internal (rather than just being internal for now to hide the public API from v8), can it be `sealed` to improve the ability of the JIT to inline?

Reply:
I think this isn't going to be hidden for a long time, so I would avoid it, however, can you expand on how sealed classes would improve things?

Reply:
I think `FaultChaosStrategy` should stay hidden though? It's implementation details of fault strategy.